src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,120 @@
+        System.out.println();
+    }
+
+    public void displayPrompt() {
+        String koreanDayOfWeek = LocalDate.now().getDayOfWeek()
+                .getDisplayName(TextStyle.FULL, Locale.KOREAN);
+        System.out.println(
+                OutputMessages.DATE_PROMPT.format(LocalDate.now().getMonthValue(), LocalDate.now().getDayOfMonth(),
+                        koreanDayOfWeek));
+        System.out.println(OutputMessages.FIRST_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.SECOND_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.THIRD_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.FOURTH_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.QUIT_FUNCTION_PROMPT.getFormat());
+        System.out.println(OutputMessages.GUIDE_PROMPT.getFormat());
+    }
+
+    public void displayCheckAttendanceResult(AttendanceRecordResponse response) {
+        int month = response.date().getMonthValue();
+        int day = response.date().getDayOfMonth();
+        String koreanDayOfWeek = response.date().getDayOfWeek()
+                .getDisplayName(TextStyle.FULL, Locale.KOREAN);
+        String formattedTime = response.time().format(DateTimeFormatter.ofPattern("HH:mm"));
+        System.out.println(DISPLAY_CHECK_ATTENDANCE_RESULT.format(month, day, koreanDayOfWeek, formattedTime,
+                response.attendanceStatus().getMessage()));
+    }
+
+    public void displayModifyAttendanceResult(ModifyAttendanceResponse response) {
+        int month = response.date().getMonthValue();
+        int day = response.date().getDayOfMonth();
+        String koreanDayOfWeek = response.date().getDayOfWeek()
+                .getDisplayName(TextStyle.FULL, Locale.KOREAN);
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("HH:mm");
+
+        String formattedOriginalTime = response.originalTime().format(dateTimeFormatter);
+        String formattedModifiedTime = response.modifiedTime().format(dateTimeFormatter);
+
+        System.out.println(DISPLAY_MODIFY_ATTENDANCE_RESULT.format(month, day, koreanDayOfWeek, formattedOriginalTime,
+                response.originalStatus(), formattedModifiedTime,
+                response.modifiedStatus()));
+    }
+
+    public void displayAttendanceRecordByName(String name, List<AttendanceRecordResponse> attendanceRecords,
+                                              TotalRecordsResponse totalRecords, String penalty) {
+        System.out.printf(DISPLAY_ATTENDANCE_RECORD_PROMPT.format(name));
+        displaySpacing();
+        for (AttendanceRecordResponse attendanceRecord : attendanceRecords) {
+            int month = attendanceRecord.date().getMonthValue();
+            int day = attendanceRecord.date().getDayOfMonth();
+            String koreanDayOfWeek = attendanceRecord.date().getDayOfWeek()
+                    .getDisplayName(TextStyle.FULL, Locale.KOREAN);
```

Comment:
여기도 `get`을 가져오고 다시 `get`하는 형태네요.
개선해봅시다!

Reply:
```suggestion
            LocalDate date = attendanceRecord.date();
            int month = date.getMonthValue();
            int day = date.getDayOfMonth();
            DayOfWeek dayOfWeek = date.getDayOfWeek();
            String koreanDayOfWeek = dayOfWeek.getDisplayName(TextStyle.FULL, Locale.KOREAN);
```

중복되는 attendanceRecord.date()까지 묶고, 
```
DayOfWeek dayOfWeek = date.getDayOfWeek();
String koreanDayOfWeek = dayOfWeek.getDisplayName(TextStyle.FULL, Locale.KOREAN);
```

이렇게 수정하면 좋을 것 같습니다!